### PR TITLE
docs(samples): refactor webhook samples

### PR DESCRIPTION
**Pull request by aribray**
Streamlines webhook samples

### CODE REVIEW

File: samples/webhook-configure-session-parameters-enable-agent-response.js [hunk 1]
```diff
@@ -13,40 +13,30 @@
 // limitations under the License.
 
 /**
- * Configures a webhook to enable an agent response.
+ * Webhook configures a session parameter to enable an agent response.
```
Comment (avallone-google) [inline samples/webhook-configure-session-parameters-enable-agent-response.js:16]: The purpose of the sample is not quite right. This is implementing Yunchen's request for "Configure webhook to configure session parameters, which will be used to generate agent responses", correct?

If so, what he means is that the agent should just set a parameter. The response will be controlled by the agent model. So, this sample just needs to set a parameter value.

Comment (avallone-google) [reply samples/webhook-configure-session-parameters-enable-agent-response.js:16]: I sent a message to Yunchen through his document to clarify.

File: samples/webhook-configure-session-parameters-trigger-transition.js [hunk 2]
```diff
@@ -19,29 +19,18 @@
...
   const targetPage = request.body.targetPage; // Must be format projects/<Project ID>/locations/<Location ID>/agents/<Agent ID>/flows/<Flow ID>/pages/<Page ID>
+
   // The value of the parameter used to trigger transition
-  let sessionParameter = request.body.sessionInfo.parameters.number;
+  const orderNumber = 123;
 
-  sessionParameter = sessionParameter *= 50;
-  const text = `We multiplied your input - the value is now ${sessionParameter}. Let's go the the next page.`;
   const jsonResponse = {
     target_page: targetPage,
```
Comment (avallone-google) [inline samples/webhook-configure-session-parameters-trigger-transition.js:29]: Based on Yunchen's response, this sample may not be needed.

File: samples/webhook-configure-session-parameters.js [hunk 3]
```diff
@@ -19,30 +19,13 @@
...
-  const tag = request.body.fulfillmentInfo.tag;
-  let newSessionParameter;
-  const text = `${newSessionParameter}. I'm a session parameter configured by the webhook. The webhook's tag is ${tag}.`;
-
-  if (tag === 'month') {
-    newSessionParameter = 'January';
-  } else if (tag === 'year') {
-    newSessionParameter = '1999';
-  }
+  // Session parameter configured by the webhook
```
Comment (avallone-google) [inline samples/webhook-configure-session-parameters.js:22]: You can't add a comment at any unchanged line of code in this tool, can you?

For line 16, change the description to "Configures a webhook to set a session parameter"

File: samples/webhook-validate-form-parameter.js [hunk 4]
```diff
@@ -19,47 +19,26 @@
...
-    },
     page_info: {
       form_info: {
         parameter_info: [
           {
-            displayName: 'paramToValidate',
+            displayName: 'orderNumber',
             required: true,
-            state: paramState,
+            state: 'VALID',
```
Comment (avallone-google) [inline samples/webhook-validate-form-parameter.js:32]: Best to use invalid for this example.

File: samples/webhook-validate-form-parameter.js [hunk 5]
```diff
@@ -19,47 +19,26 @@
...
           },
         ],
       },
     },
     sessionInfo: {
       parameters: {
-        // Set session parameter to null if your agent needs to reprompt the user
-        paramToValidate: paramToValidate,
+        // Set session parameter to null if the form parameter is 'INVALID' and your agent needs to reprompt the user
+        orderNumber: 123,
```
Comment (avallone-google) [inline samples/webhook-validate-form-parameter.js:41]: This is not getting set to null.

**Review by avallone-google — APPROVED** [review]

**Review by kweinmeister — APPROVED** [review]

**Pull request merged**